package.json

```diff
@@ -132,16 +115,14 @@
     "jest-enzyme": "7.0.x",
     "lint-staged": "8.1.0",
     "lodash": "4.x.x",
-    "react-docgen-typescript-loader": "3.0.x",
     "rimraf": "2.6.x",
     "semantic-release": "15.12.5",
     "shelljs": "0.8.x",
     "ts-jest": "23.10.5",
     "ts-loader": "5.3.1",
```

Comment:
I guess we are good to remove it, right?

Reply:
Yup!